packages/open-next/src/build/compileCache.ts

```diff
@@ -15,19 +15,49 @@ export function compileCache(
 ) {
   const { config } = options;
   const ext = format === "cjs" ? "cjs" : "mjs";
-  const outFile = path.join(options.buildDir, `cache.${ext}`);
+  const compiledCacheFile = path.join(options.buildDir, `cache.${ext}`);
+  const compiledComposableCacheFile = path.join(
```

Comment:
nits:
- update the function  comments line 6
- move `const compiledComposableCacheFile` to l 52